src/search/services/search.service.ts

```diff
@@ -37,6 +39,26 @@ export class SearchService {
     return window.sessionStorage.getItem(key) ? JSON.parse(window.sessionStorage.getItem(key)) : null;
   }
 
+  public decrementStartRecord(): void {
+    const startRecord = this.retrieveState(SearchStatePersistenceKey.START_RECORD);
+    if (startRecord) {
+      const startRecordNumber = parseInt(startRecord, 10);
+      if (startRecordNumber > this.MAX_RECORD_PAGE_SIZE) {
+        this.storeState(SearchStatePersistenceKey.START_RECORD, startRecordNumber - this.MAX_RECORD_PAGE_SIZE);
+      }
+    }
+  }
+
+  public incrementStartRecord(): void {
```

Comment:
This should return the new start record - because where ever it is used, the first thing you do is query it

Reply:
@andywilkinshmcts - as above... it had crossed my mind.